website/docs/zh/guide/tech/json.mdx

```diff
@@ -10,5 +12,20 @@ JSON 是 Rspack 的一等公民，你可以直接导入，例如：
 
 ```ts title="index.js"
 import json from './example.json';
-json.foo; // bar
+console.log(json.foo); // "bar"
+```
+
+## 具名引用
```

Comment:
```suggestion
## 具名导入
```

Reply:
Whether you believe or not, I purposely aligned it with Rsbuild 🤣. https://rsbuild.dev/zh/guide/basic/json-files#%E5%85%B7%E5%90%8D%E5%BC%95%E7%94%A8